src/solvers/dgsem_tree/cache_viscous.jl

```diff
@@ -0,0 +1,33 @@
+mutable struct CacheViscous1D{uEltype <: Real}
```

Comment:
This is similar in structure to our other `Container` types, isn't it? Maybe you can call it `ViscousContainer` (or `ViscousDataContainer` - not sure) and then just refer to it in the cache as `viscous_data`?

Since we have this very strong notion of one cache per semi to rule them all, I would caution to not use a hierarchy of things called "cache", thus I recommend to get rid of the "cache" part in both the type and the variable under which it is stored.

Reply:
Good idea - yeah, this draws its purpose from the containers.